test/test_meta.py

```diff
@@ -752,8 +768,14 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
     aten.empty.memory_format: {b8, bf16, c128, c64, c32, f16, f32, f64, i16, i32, i64, i8, u8},
     aten.bucketize.Tensor : {f16, i8, f64, i64, bf16, f32, i32, i16, u8},
     aten.bucketize.Tensor_out : {f16, i8, f64, i64, bf16, f32, i32, i16, u8},
+    aten.addbmm_.default: {bf16, c128, c64, f32, f64, i16, i32, i64, i8, u8},
 }
 
+# For CompositeImplicitAutograd functions that fail before hitting the Mode
```

Comment:
what exactly does this mean?

Reply:
For float_power_, the composite implicit implementation will fail with something like 'the base given to float_power_ has dtype Byte but the operation's result requires dtype Double'. But since that happens before we hit the DispatchMode. That means that all the skips above (which happen inside the mode) can't be used and we need one that skips running that function altogether.
Does that make sense?